domain/tutoringModel/generate_learning_path.py

```diff
@@ -0,0 +1,109 @@
+from collections import OrderedDict
+import pytest
+
+
+class GenerateLearningPath:
```

Comment:
The class should be for Learning Path in general and not specific for generation since we probably will add more things to the class